src/index.ts

```diff
@@ -265,14 +265,18 @@ export async function create(appName: string, options: Options) {
   })
 
   // create LICENSE
-  const license = makeLicenseSync(args.license, {
-    year,
-    project: name,
-    description: args.description,
-    organization: getContact(args.author, args.email),
-  })
-  const licenseText = license.header + license.text + license.warranty
-  fs.writeFileSync(path.resolve(packageDir, 'LICENSE'), licenseText)
+  try {
+    const license = makeLicenseSync(args.license, {
+      year,
+      project: name,
+      description: args.description,
+      organization: getContact(args.author, args.email),
+    })
+    const licenseText = license.header + license.text + license.warranty
+    fs.writeFileSync(path.resolve(packageDir, 'LICENSE'), licenseText)
+  } catch (e) {
+    // do not generate LICENSE
```

Comment:
What kind of errors are you expecting from this block?

Reply:
`makeLicenseSync()` throws "License not found" error when unexpected license is chosen https://github.com/sergeysova/license.js/blob/1b0b605cd895230df54c15e24120cecb59c82263/lib/index.js#L12-L14
`UNLICENSED` is "unexpected license" for license.js npm. https://github.com/sergeysova/license.js/blob/master/licenses/index.json